src/pybind/mgr/dashboard/frontend/src/app/ceph/shared/smart-list/fixtures/smart_data_version_1_0_hdd_scsi_response.json

```diff
@@ -0,0 +1,207 @@
+          "Transport protocol:   SAS (SPL-3)",
+          "Local Time is:        Sun May  8 14:21:11 2022 UTC",
+          "SMART support is:     Available - device has SMART capability.",
+          "SMART support is:     Enabled",
+          "Temperature Warning:  Enabled",
+          "Read Cache is:        Enabled",
+          "Writeback Cache is:   Enabled",
+          "",
+          "=== START OF READ SMART DATA SECTION ===",
+          "SMART Health Status: OK",
+          "",
+          "Grown defects during certification <not available>",
+          "Total blocks reassigned during format <not available>",
+          "Total new blocks reassigned <not available>",
+          "Power on minutes since format <not available>",
+          "Current Drive Temperature:     38 C",
+          "Drive Trip Temperature:        85 C",
+          "",
+          "Manufactured in week 43 of year 2021",
+          "Specified cycle count over device lifetime:  50000",
+          "Accumulated start-stop cycles:  9",
+          "Specified load-unload count over device lifetime:  600000",
+          "Accumulated load-unload cycles:  74",
+          "Elements in grown defect list: 0",
+          "",
+          "Error counter log:",
+          "           Errors Corrected by           Total   Correction     Gigabytes    Total",
+          "               ECC          rereads/    errors   algorithm      processed    uncorrected",
+          "           fast | delayed   rewrites  corrected  invocations   [10^9 bytes]  errors",
+          "read:          0        0         0         0       1001       8519.006           0",
+          "write:         0        0         0         0      25720     146241.629           0",
+          "verify:        0        0         0         0        261          0.000           0",
+          "",
+          "Non-medium error count:        0",
+          "",
+          "No Self-tests have been logged",
+          "",
+          "Background scan results log",
+          "  Status: waiting until BMS interval timer expires",
+          "    Accumulated power on time, hours:minutes 1719:55 [103195 minutes]",
+          "    Number of background scans performed: 5,  scan progress: 0.00%",
+          "    Number of background medium scans performed: 5",
+          "",
+          "Protocol Specific port log page for SAS SSP",
+          "relative target port id = 1",
+          "  generation code = 3",
+          "  number of phys = 1",
+          "  phy identifier = 0",
+          "    attached device type: expander device",
+          "    attached reason: loss of dword synchronization",
+          "    reason: unknown",
+          "    negotiated logical link rate: phy enabled; 12 Gbps",
+          "    attached initiator port: ssp=0 stp=0 smp=1",
+          "    attached target port: ssp=0 stp=0 smp=1",
+          "    SAS address = 0x5000cca2c3d5f545",
+          "    attached SAS address = 0x5000ccab041140bf",
```

Comment:
Could these be unique IDs and/or sensitive info?

Reply:
Changed them! thanks